.github/workflows/release-github-template.yml

```diff
@@ -1,56 +1,28 @@
-name: "Release on Github"
+name: "Release to github registry"
 
 on:
-  workflow_call:
+  push:
```

Comment:
reusable workflow must have
```
 on:
  workflow_call:
```

You forget also `publish-to-github-maven-registry` input

You can check what we did in [2.7.0.1](https://github.com/chutney-testing/chutney/blob/2.7.0.1/.github/workflows/release-github-template.yml) tag